src/BasisTheory.net/Tokens/Entities/EncryptionKey.cs

```diff
@@ -12,5 +12,9 @@ public class EncryptionKey
         [JsonProperty("alg")]
         [JsonPropertyName("alg")]
         public string Algorithm { get; set; }
+
+        [JsonProperty("prov")]
+        [JsonPropertyName("prov")]
```

Comment:
Is `prov` a well-known abbreviation of "provider"? This may be hard to understand by just looking at the JSON payload.

Reply:
Looks like it is. I wasn't familiar.